README.md

```diff
@@ -20,28 +24,69 @@ Or install it yourself as:
 
     $ gem install ssm_config
 
+## Setup
+
+To utilize ActiveRecords, create the following model:
+```
+rails generate model SsmConfigRecord file:string accessor_keys:string value:string
```

Comment:
We should probably add an index in the migration on the `file` column for ease of query. Might be OK to just list the whole migration in here after the generate command.

Reply:
Do you mean something like `file:string:index`?